tests/acceptance/features/coreApiWebdavPreviews/previews.feature

```diff
@@ -9,146 +9,238 @@ Feature: previews of files downloaded through the webdav API
-    Given user "Alice" has uploaded file "filesForUpload/testavatar.jpg" to "/testimage.jpg"
+  Scenario Outline: download previews of image after renaming it
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file "filesForUpload/testavatar.jpg" to "/testimage.jpg"
     And user "Alice" has moved file "/testimage.jpg" to "/testimage.txt"
     When user "Alice" downloads the preview of "/testimage.txt" with width "32" and height "32" using the WebDAV API
     Then the HTTP status code should be "200"
     And the downloaded image should be "32" pixels wide and "32" pixels high
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
 
 
   Scenario Outline: download previews of shared files (to shares folder)
-    Given user "Brian" has been created with default attributes and without skeleton files
+    Given using <dav-path-version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
     And user "Alice" has uploaded file "filesForUpload/<resource>" to "/<resource>"
     And user "Alice" has shared file "/<resource>" with user "Brian"
     And user "Brian" has accepted share "/<resource>" offered by user "Alice"
     When user "Brian" downloads the preview of "/Shares/<resource>" with width "32" and height "32" using the WebDAV API
     Then the HTTP status code should be "200"
     And the downloaded image should be "32" pixels wide and "32" pixels high
     Examples:
-      | resource    |
-      | lorem.txt   |
-      | example.gif |
+      | dav-path-version | resource    |
+      | old              | lorem.txt   |
+      | old              | example.gif |
+      | new              | lorem.txt   |
+      | new              | example.gif |
 
 
-  Scenario: download previews of other users files
-    Given user "Brian" has been created with default attributes and without skeleton files
+  Scenario Outline: user tries to download previews of other users files
+    Given using <dav-path-version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
     And user "Alice" has uploaded file "filesForUpload/lorem.txt" to "/parent.txt"
     When user "Brian" downloads the preview of "/parent.txt" of "Alice" with width "32" and height "32" using the WebDAV API
     Then the HTTP status code should be "404"
-    And the value of the item "/d:error/s:message" in the response about user "Alice" should be "File not found: parent.txt in '%username%'"
+    And the value of the item "/d:error/s:message" in the response about user "Alice" should be "File with name parent.txt could not be located"
     And the value of the item "/d:error/s:exception" in the response about user "Alice" should be "Sabre\DAV\Exception\NotFound"
+    Examples:
+      | dav-path-version |
+      | new              |
+      | spaces           |
```

Comment:
```suggestion
    Examples:
      | dav-path-version |
      | new              |
      | old              |
      | spaces           |
```
old also missing? or ?

Reply:
using old dav endpoint, we won't know which user endpoint we are requesting to.